src/core/libmaven/Scan.h

```diff
@@ -29,6 +29,17 @@ class Scan
     */
     inline unsigned int nobs() const { return mz.size(); }
 
```

Comment:
Is the m/z array sorted? or is that an assumption we're making

Reply:
An assumption, which, now that I think of it, we should not be making. Returning the `std::min` and `std::max` of `mz` vector is definitely the right way of doing this.